src/components/NavigatorSearch/modes/messages.ts

```diff
@@ -101,6 +101,14 @@ const messages = defineMessages({
     id: "x/ZVlU",
     defaultMessage: "Product",
   },
+  root: {
+    defaultMessage: "This is root category",
```

Comment:
suggestion: let's add `context` for this text, for example:

```suggestion
    defaultMessage: "This is root category",
    context: "Navigation search",
```